sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -112,12 +114,24 @@ describe("RequestResponseLink", function() {
           body: "Hello World!!"
         }
       });
-    }, 4000);
-    const response = await link.sendRequest(request, {
-      delayInSeconds: 1,
-      timeoutInSeconds: 5
-    });
-    assert.equal(response.correlation_id, messageId);
+    }, 1000);
+
+    const sendRequestPromise = async () => {
+      await link.sendRequest(request, {
+        timeoutInSeconds: 5
+      });
+    };
+
+    const config: RetryConfig<void> = {
+      operation: sendRequestPromise,
+      connectionId: "connection-1",
+      operationType: RetryOperationType.management,
+      maxRetries: 3,
+      delayInSeconds: 1
+    };
+
+    await retry<void>(config);
```

Comment:
Can we also check the message that is actually returned here?
I believe the first attempt returns a message no body and the successful attempt returns the body `Hello World!!`
The below should work right?

```suggestion
    const message = await retry<void>(config);
    assert.equal(message.body, "Hello World!!");
    
```

Reply:
Makes sense, updated test